### ISSUE TITLE
Empty view Y center shift in UICollectionView 

### ISSUE THREAD

**Issue by alphatroya** [eilianlove/EmptyKit#4]
When UICollectionView has it's own non-zero `contentInset`, the vertical centre of the empty view is moving right according to left edge indent

<img width="423" alt="image" src="https://cloud.githubusercontent.com/assets/4927633/26103503/7c2a9914-3a42-11e7-931f-c5e2eaabd3fd.png">
.

**Comment by ZionChang** [comment]
Could you please send me a demo or some code?

**Comment by alphatroya** [comment]
The source code is simple. I just create UICollectionView instance and set-up content inset. When I comment content inset line, centring is fine

``` swift

let flowLayout = UICollectionViewFlowLayout()
let indent: CGFloat = 20.0
flowLayout.minimumLineSpacing = indent
flowLayout.minimumInteritemSpacing = indent

let collectionView = UICollectionView(frame: CGRect.zero, collectionViewLayout: flowLayout)
collectionView.dataSource = self
collectionView.delegate = self
collectionView.contentInset = UIEdgeInsets(top: indent, left: indent, bottom: indent, right: indent)
collectionView.ept.dataSource = self
collectionView.ept.delegate = self

```

**Comment by ZionChang** [comment]
Thanks for your issue!
It should be fixed in the new version(3.1.1).
I will close the issue for now.
